docs/blog/introduction-to-graphql.md

```diff
@@ -13,14 +13,14 @@ Broadly speaking an API can be defined as a way for programmers to interface wit
 
 ## Rest APIs
 
-The traditional method for interfacing with applications on the web is a REST API. One defining characteristic of REST APIs is they use [HTTP request methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), such as `GET`, `PUT`, `POST` &amp;`DELETE` to "CRUD" operations. Typically, `PUT` for Create, `GET` for Read, `POST` for Update and `DELETE` for Delete.
+The traditional method for interfacing with applications on the web is a REST API. One defining characteristic of REST APIs is they use [HTTP request methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), such as `GET`, `PUT`, `POST` &`DELETE` to "CRUD" operations. Typically, `PUT` for Create, `GET` for Read, `POST` for Update and `DELETE` for Delete.
```

Comment:
I ran `yarn format` and this change came out. Might as well include this here for now.